src/server/utils/css.ts

```diff
@@ -59,4 +59,41 @@ export default `
     white-space: normal;
   }
   
+
+.delvilkår {
+  margin-bottom: 5px;
+}
+
+.vilkårsresultat {
+  font-size: 1.2rem;
+  margin-bottom: 16px;
+  margin-top: 10px;
+}
+
+.page-break {
+  page-break-inside: avoid;
+}
+
+.vilkårsresultat-ikon {
+  display: inline-block;
+  width: 24px;
+  height: 24px;
+  position: relative;
+  top: 5px;
+  margin:0;
+  padding:0;
+}
+
+h3, h4 {
+  margin-bottom: 3px;
+}
+
+table, th, td {
+border: 1px solid black;
+border-collapse: collapse;
```

Comment:
@halvorbmundal Kan denne skape "trøbbel" for dere? Bruker dere tabeller i brevene, og isåfall - er det med border? 🤔

Reply:
Stylingen som settes på andre generelle html-elementer som h3 og h4 kan vel også skape litt layout-shift her?